snapcraft/main.py

```diff
@@ -121,6 +122,7 @@
   status       Show the current status of a snap per channel and architecture.
   list-revisions List all revisions of a snap.
   revisions    Alias for list-revisions.
+  history      List all revisions of a snap.
```

Comment:
These descriptions need to be spaced out as well. Also, instead of a description for history, I'd say "Deprecated alias for list-revisions" or something similar.